doc/source/index.rst

```diff
@@ -205,6 +205,9 @@ exceptions, as in the cases here.
 
 .. testcode::
 
+    class ClientError(Exception):
+        """Some type of client error."""
+
```

Comment:
I'd put that in a different PR though.

Reply:
I would, too... but checks fail without this in place. I can send this commit in first.

Reply:
It's ok, we'll rebase it.